spec/support/shoulda_matchers.rb

```diff
@@ -0,0 +1,13 @@
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :minitest
+    with.library :rails
+  end
+end
```

Comment:
this is not necessary. This config block is for when you are using minitest as the test runner/suite. In our case, we are using rspec. Also, remember to add the `# frozen_string_literal: true` magic comment on top of the file 😄